src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -292,6 +292,7 @@ private static SyntaxKind GetTokenKind(OperatorKind kind)
                 OperatorKind.Multiply => SyntaxKind.AsteriskToken,
                 OperatorKind.OnesComplement => SyntaxKind.TildeToken,
                 OperatorKind.RightShift => SyntaxKind.GreaterThanGreaterThanToken,
+                OperatorKind.UnsignedRightShift => SyntaxKind.GreaterThanGreaterThanGreaterThanToken,
```

Comment:
Another scenario that may be worth validating is MetadataAsSource. Hopefully, it's using the same syntax generator and is fine.